docker/httpd/Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN apt-get update && apt-get install -y \
 RUN mkdir -p /usr/local/bin
 COPY lmutil /usr/local/bin/
 RUN chmod 755 /usr/local/bin/lmutil
+# older lmutil need this
+RUN ln -s /lib64/ld-linux-x86-64.so.2 /lib64/ld-lsb-x86-64.so.3
```

Comment:
I can confirm that this symlink is needed for a Debian 11 environment.  Unfortunately, I get the following error when building the container...
```
 => ERROR [php-httpd  9/18] RUN ln -s /lib64/ld-linux-x86-64.so.2 /lib64/ld-lsb-x86-64.so.3                        0.2s
------                                                                                                                  
 > [php-httpd  9/18] RUN ln -s /lib64/ld-linux-x86-64.so.2 /lib64/ld-lsb-x86-64.so.3:
0.154 ln: failed to create symbolic link '/lib64/ld-lsb-x86-64.so.3': File exists
------
failed to solve: process "/bin/sh -c ln -s /lib64/ld-linux-x86-64.so.2 /lib64/ld-lsb-x86-64.so.3" did not complete successfully: exit code: 1
```

Reply:
My testing has found that this issue is caused by a file conflict with line 11:
https://github.com/rpi-dotcio/phpLicenseWatcher/blob/5fbf0a01c186476c8553cac75119d46b8f9399ea/docker/httpd/Dockerfile#L11

By the time we get to line 24, the symlink already exists and the build is forced to abort.
https://github.com/rpi-dotcio/phpLicenseWatcher/blob/5fbf0a01c186476c8553cac75119d46b8f9399ea/docker/httpd/Dockerfile#L24

My testing showed that rewriting line 11 as line 24 (and removing line 24) fixes the issue.